src/azure-cli/azure/cli/command_modules/databoxedge/linter_exclusions.yml

```diff
@@ -0,0 +1,10 @@
+databoxedge device storage-account create:
+    parameters:
+        storage_account_credential_id:
+            rule_exclusions:
+            - option_length_too_long
```

Comment:
Can you add a short option for this parameter instead of adding rule exclusion?

Reply:
Done